input/create_bot.py

```diff
@@ -1,22 +1,27 @@
-
 import json
+import os
 from os.path import exists
-keys = ["bot_name", "bot_size","bot_prey","bot_predator","bot_reproduction_rate","bot_adult_age"]
+keys = ["bot_name", "bot_size", "bot_prey", "bot_predator", "bot_reproduction_rate", "bot_adult_age"]
+
+
 def create_json(file_name,data):
-    if exists(file_name) == False:
-        with open(file_name, 'w') as file:
+    if exists(os.path.join(os.getcwd(), "bots", file_name)) == False:
+        with open((os.path.join(os.getcwd(), "bots", file_name)), 'w') as file:
```

Comment:
don't repeat the same thing twice. you should've set a variable like `file_path = os.path.join(os.getcwd(), "bots",file_name)` and then used that